src/rules/no-same-line-conditional.ts

```diff
@@ -70,6 +73,20 @@ const rule: TSESLint.RuleModule<string, string[]> = {
             {
               messageId: 'sameLineCondition',
               loc: followingIfToken.loc,
+              suggest: [
+                {
+                  messageId: 'suggestAddingElse',
+                  fix: fixer => fixer.insertTextBefore(followingIfToken, 'else '),
+                },
+                {
+                  messageId: 'suggestAddingNewline',
+                  fix: fixer =>
+                    fixer.insertTextBefore(
+                      followingIf,
+                      '\n' + ' '.repeat(precedingIf.range[0] - 1),
```

Comment:
i am not sure it is correct to use range here, I think it should be loc (add test with not first statement)

Reply:
How silly of me. It should definitely be based on loc.